components/console-backend-service/internal/gqlschema/schema.graphql

```diff
@@ -26,7 +26,7 @@ input ResourceAttributes {
     isChildResolver: Boolean! = false
 }
 
-# Rafter & Asset Store & CMS
+# Rafter
```

Comment:
There is no mention about rafter in the schema. this comment is just exposing the implementation and should be removed.

Reply:
This comment point that below types reference to Rafter resources, but as I wrote in another comment, for me it makes no difference.

Reply:
I agree that domain can/should be named after k8s `apiGroup` as you mentioned above. However, I don't think this information should leak to the API. It's not a blocker, though.